policy/modules/admin/readahead.te

```diff
@@ -42,7 +42,6 @@ dev_getattr_generic_blk_files(readahead_t)
 dev_getattr_all_chr_files(readahead_t)
 dev_getattr_all_blk_files(readahead_t)
 dev_dontaudit_read_all_blk_files(readahead_t)
-dev_dontaudit_getattr_memory_dev(readahead_t)
```

Comment:
I don't think this `dontaudit` should be removed.

Reply:
Restored